test/fixtures/mock-sdk/lib/toolkit/loader.js

```diff
@@ -759,14 +769,14 @@ const Loader = iced(function Loader(options) {
     load: { enumerable: false, value: options.load || load },
     // Main (entry point) module, it can be set only once, since loader
     // instance can have only one main module.
-    main: new function() {
+    main: function() {
```

Comment:
I have no idea why this was using new. Some obscure trick maybe? Should we wrap it in an ignore?